### DISCUSSION TITLE
Standup (Thursday, April 28, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#26] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #21 
 - Finished #17 

**What are your priorities for the next 24 hours?**:
 - Planning
 - Take a new issue
 - Create fake projectc to show architecture

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto for being so great at Figma

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Established work groups and tech stack with everyone

**What are your priorities for the next 24 hours?**:
 - Take a new task

**Blockers**:
 -

**Shoutouts**:
 - @CarlosCRG19 for recording the video

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 -  Finished Sign Up mockup #21 😀
 - Nothing else, I think

**What are your priorities for the next 24 hours?**:
 - Start working on actual code. Pretty excited 😁
 - Also, would be nice to define some tasks to start coding 

**Blockers**:
 - We do not have tasks yet

**Shoutouts**:
 - @CarlosCRG19 for recording our mockup demo and @BrandonMagana for turning sprint 0's assignment in 😍

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #12 and #22 

**What are your priorities for the next 24 hours?**:
 - Make some research on PERN
 - Take a new task


**Shoutouts**:
 - @CarlosCRG19 for making the video and @BrandonMagana for tuning the document.

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - nothing

**What are your priorities for the next 24 hours?**:
 - Planning 
 - Take a new task
**Blockers**:
 -

**Shoutouts**:
@CarlosCRG19  @CristianCazares @antonieto @DavBot02 @BrandonMagana

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Not that much, just turned in the document

**What are your priorities for the next 24 hours?**:
 -  Learn more about the tech stack
 - Define tasks

**Blockers**:
 - Problems installing postgresql

**Shoutouts**:
- @CarlosCRG19  for recording the video
